pkg/services/accesscontrol/models.go

```diff
@@ -275,7 +275,7 @@ type SetResourcePermissionCommand struct {
 
 type SaveExternalServiceRoleCommand struct {
 	OrgID             int64
-	Global            bool
+	Global            bool // global assignment of the role
```

Comment:
I'm considering removing this all together after @volcanonoodle challenged the fact that we'd have no use case for it.
Also given we now consider `OrgID` to be solely the assignment's org, should I rename the `OrgID` to `AssignmentOrgID`?